src/stores/appcatalog/selectors.ts

```diff
@@ -28,5 +28,8 @@ export function selectIngressAppFromCluster(cluster: Cluster) {
 export function selectReadmeURL(
   appVersion: IAppCatalogApp
 ): string | undefined {
-  return appVersion.sources?.find((url) => url.endsWith(Constants.README_FILE));
+  return (
+    appVersion.annotations?.['application.giantswarm.io/readme'] ||
+    appVersion.sources?.find((url) => url.endsWith(Constants.README_FILE))
```

Comment:
Neither one of `annotations` or `sources` are optional in the interface.
https://github.com/giantswarm/happa/blob/f58409e8798101eb8d712d43f2da71ad2914e9cd/src/%40types/appcatalogs.d.ts#L18

Should they be?

Reply:
Yup! They could be there or not there depending on the app. Our own apps should always have it, but it's still something that we could forget on accident. Fixing 👍